packages/crud/test/a11y.test.js

```diff
@@ -256,4 +256,70 @@ describe('a11y', () => {
       </vaadin-crud>
   `),
   );
+
+  ['aside', 'bottom'].forEach((editorPosition) => {
+    describe(`create item - editor position ${editorPosition}`, () => {
+      let newButton, scroller;
+
+      beforeEach(async () => {
+        crud = fixtureSync('<vaadin-crud></vaadin-crud>');
+        crud.editorPosition = editorPosition;
+        crud.items = [{ title: 'Item 1' }];
+        await nextRender();
+        newButton = crud.querySelector('[slot=new-button]');
+        scroller = crud.$.scroller;
+      });
+
+      afterEach(async () => {
+        crud.editorOpened = false;
+        await nextRender();
+      });
+
+      it(`should set tabindex on the scroller when using editor position ${editorPosition}`, async () => {
```

Comment:
This test is probably superfluous since the next test already verifies the presence of tabindex. You cannot programmatically focus an unfocusable element unless it has a tabindex.

Reply:
Makes sense, thanks. Removed tests checking `tabindex` presence.